src/status_im/contexts/wallet/add_account/add_address_to_save/view.cljs

```diff
@@ -0,0 +1,82 @@
+(ns status-im.contexts.wallet.add-account.add-address-to-save.view
+  (:require
+    [clojure.string :as string]
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [status-im.common.floating-button-page.view :as floating-button-page]
+    [status-im.contexts.wallet.add-account.add-address-to-save.style :as style]
+    [status-im.contexts.wallet.common.activity-indicator.view :as activity-indicator]
+    [status-im.contexts.wallet.common.address-input.view :as address-input]
+    [status-im.contexts.wallet.utils :as wallet.utils]
+    [status-im.subs.wallet.add-account.address-to-watch]
+    [utils.ens.core :as utils.ens]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn- on-press-confirm-add-address-to-save
+  [input-value]
+  (rf/dispatch
+   [:wallet/confirm-add-address-to-save
+    {:address input-value
+     :ens?    (utils.ens/is-valid-eth-name?
+               input-value)}]))
+
+(defn view
+  []
+  (let [addresses (rf/sub [:wallet/lowercased-saved-addresses])
+        {:keys [title description input-title accessibility-label]}
+        (rf/sub [:wallet/currently-added-address])
+        validate #(wallet.utils/validate-fn % addresses)
+        customization-color (rf/sub [:profile/customization-color])]
+    (wallet.utils/clear-activity-and-scanned-address)
+    (fn []
+      (let [activity-state                          (rf/sub [:wallet/watch-address-activity-state])
+            validated-address                       (rf/sub [:wallet/watch-address-validated-address])
+            [input-value set-input-value]           (rn/use-state nil)
+            [validation-msg set-validation-message] (rn/use-state nil)
+            clear-input                             (fn []
+                                                      (set-input-value nil)
+                                                      (set-validation-message nil)
+                                                      (wallet.utils/clear-activity-and-scanned-address))]
```

Comment:
Use state hooks instead of ratoms.

Reply:
I'm not sure, but I think we're not using the form-2 style components either (?)